Unix/installbuilder/datafiles/Base_OMI.data

```diff
@@ -511,10 +518,20 @@ fi
 %Postinstall_20
 #include PAM_Functions
 ConfigurePAM
+chown omi:omi /var/opt/omi/log
+chown omi:omi /var/opt/omi/run
+chown omi:omi /etc/opt/omi/ssl/omikey.pem
 
 # Fix potential permissons issue on /etc/opt/omi directory
 chown root:${{ROOT_GROUP_NAME}} /etc/opt/omi
 
 %Preuninstall_100
 #include PAM_Functions
 UnconfigurePAM
+
+%Postuninstall_90
+egrep "^omi:" /etc/passwd 2>&1 >/dev/null
+if [ $? -eq 0 ]; then
+    echo "Deleting omi service account ..."
+    userdel omi
```

Comment:
Full path is a general issue, and we're not consistent in our data files.  Let's discuss this further.